engine/src/flutter/runtime/runtime_controller.h

```diff
@@ -767,7 +769,7 @@ class RuntimeController : public PlatformConfigurationClient,
               double height) override;
 
   // |PlatformConfigurationClient|
-  void UpdateSemantics(SemanticsUpdate* update) override;
+  void UpdateSemantics(SemanticsUpdate* update, int64_t view_id) override;
```

Comment:
Nit: view id before the update if we keep these separate.

```suggestion
  void UpdateSemantics(int64_t view_id, SemanticsUpdate* update) override;
```